src/main/java/org/entando/entando/web/common/exceptions/ValidationUnprocessableEntityException.java

```diff
@@ -0,0 +1,16 @@
+package org.entando.entando.web.common.exceptions;
+
+import org.springframework.validation.BindingResult;
+
+public class ValidationUnprocessableEntityException extends RuntimeException {
```

Comment:
Correct english name would be `UnprocessableEntityValidationException` but I see we have this naming convention in existing classes :(